### ISSUE TITLE
The default etcd address can't be parsed correctly on the newest go version

### ISSUE THREAD

**Issue by freesky-edward** [coreos/torus#444]
I am not sure whether it is a bug of torus or golang itself. I file a bug here for deep discussion.
**my case**
when I run the cmd "torusctl init" with the newest go version, I got the following error message:
    start to init MDS...
    MDS init finished...
    invalid etcd address: parse 127.0.0.1:2379: first path segment in URL cannot contain colon
**root cause**
I found the golang community changed [its url.Parse method](https://go-review.googlesource.com/#/c/31582/) after I research into the code. they think "127.0.0.1:2379" is no longer the correct address according to RFC 3986. it raises the error message from the "flagconfig.BuildConfigFromFlags" when parse the "defaultEtcdAddress" via url.Parse

**Comment by mischief** [comment]
try using http://127.0.0.1:2379

**Comment by freesky-edward** [comment]
yeah, it  works well. should we change the defaultEtcdAddress value of flagconfig to "http://127.0.0.1:2379"?
